pkg/cliplugins/bind/cmd/cmd.go

```diff
@@ -0,0 +1,70 @@
+/*
+Copyright 2022 The KCP Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+
+	"github.com/kcp-dev/kcp/pkg/cliplugins/bind/plugin"
+)
+
+var (
+	// TODO: add other examples related to permission claim commands.
+	bindExampleUses = `
+	# Create an APIBinding named "my-binding" that binds to the APIExport "my-export" in the "root:my-service" workspace.
+	%[1]s bind apiexport root:my-service:my-export --name my-binding
```

Comment:
Can `APIBinding.metadata.name` have a colon in it? Are we OK making this type of concatenation a user-facing thing?

Reply:
Based on k8s documentation, both the RFCs allow only alphanumeric with "-" as object names (https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-label-names). The approach of having <ws_path>:<object_name> was one of the design discussion: https://docs.google.com/document/d/1J31wXY1-2aCyyGFjUlusKoHDzV-zktAmRdIMjMf4OR0/edit?disco=AAAAbWW6NnA